### PR TITLE
Update nft-tools.mdx to add Crossmint as one of the NFT Tools

### LINKED ISSUE #333: [DOCS] Add Crossmint as one of the NFT Tools
## Brief Description of the Docs Request
Crossmint is self-serve on Optimism Mainnet and Testnet. Crossmint serves to provide simple and scalable Minting APIs for NFTs to be created on Optimism. Crossmint also powers NFT Checkouts which enables credit card and cross-chain payments for NFTs. 

## Description of the Documentation You'd Like
Add Crossmint to NFT Tools sections on the Dev docs

## Links to Related Pages
http://crossmint.com/

## Additional Context
Will explain in the PR created

### PR DESCRIPTION
**Pull request by rohit-710**
**Description**

- Listed Crossmint as one of the available tools
- Updated the feature comparison table to include Crossmint

**Tests**

- No test added, checked if the content was formatted using a simple preview

**Additional context**

Crossmint is self-serve on Optimism Mainnet and Testnet. Crossmint serves to provide simple and scalable Minting APIs for NFTs to be created on Optimism. Crossmint also powers NFT Checkouts which enables credit card and cross-chain payments for NFTs. 

You can learn more about Crossmint [here](http://crossmint.com/)

**Metadata**

- Fixes #333

### CODE REVIEW

**Comment by rohit-710** [comment]
@coderabbitai resume

File: pages/builders/tools/nft-tools.mdx [hunk 1]
```diff
@@ -25,24 +25,25 @@ These tools are available on OP Mainnet:
 *   [Zero Code NFT](https://www.zerocodenft.com/)
 *   [Unlock](https://unlock-protocol.com/) (time-bound NFTs for membership)
 *   [ThirdWeb](https://thirdweb.com/)
+*   [Crossmint](http://crossmint.com/?utm_source=backlinks&utm_medium=docs&utm_campaign=optimism)
```
[the remark this answers is not in the document]
Comment (rohit-710) [reply pages/builders/tools/nft-tools.mdx]: @coderabbitai resume

**Review by rohit-710 — COMMENTED** [review]
@coderabbitai resume

**Comment by rohit-710** [comment]
Hey @sbvegan 
Can you please review this PR?

**Review by smartcontracts — APPROVED** [review]

**Comment by smartcontracts** [comment]
Thanks!

**Pull request merged**